src/cljs/rems/atoms.cljs

```diff
@@ -78,18 +78,19 @@
   (let [wrapped-on-change (fn [e]
                             (.preventDefault e)
                             (.stopPropagation e)
-                            (when on-change
-                              (on-change value)))]
+                            (on-change value))]
     [:i.far.fa-lg
-     {:id id
-      :class [:checkbox class (if value :fa-check-square :fa-square)]
-      :tabIndex 0
-      :role :checkbox
-      :aria-checked value
-      :aria-label (if value (text :t.form/checkbox-checked) (text :t.form/checkbox-unchecked))
-      :on-click wrapped-on-change
-      :on-key-press #(when (= (.-key %) " ")
-                       (wrapped-on-change %))}]))
+     (cond-> {:id id
+              :class [:checkbox class (if value :fa-check-square :fa-square)]
+              :tabIndex 0
+              :role :checkbox
+              :aria-checked value
+              :aria-label (if value
+                            (text :t.form/checkbox-checked)
+                            (text :t.form/checkbox-unchecked))}
+       on-change (assoc :on-click wrapped-on-change
+                        :on-key-press #(when (= (.-key %) " ")
+                                         (wrapped-on-change %))))]))
```

Comment:
Should we `let` the `wrapped-on-change` here if it's not used elsewhere?

Reply:
If you like, you can fix this later, I'll merge to get the fix in.

Reply:
did you mean binding inside the conditional expression on line 92? the `cond->` tries to macroexpand the expression so that cannot work

Reply:
Yeah, right, that would make it rather difficult to thread in.